packages/kbn-es-query/src/es_query/index.d.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export function buildQueryFromFilters(filters: unknown[], indexPattern: unknown): unknown;
+export function buildEsQuery(
+  indexPattern: unknown,
+  queries: unknown,
+  filters: unknown,
```

Comment:
One cannot access properties or elements from an [unknown type](https://github.com/Microsoft/TypeScript/pull/24439). Since queries and filters are now typed as unknown, will this not effect `buildEsQuery` implementations when we need to [access properties from either a filter or a query](https://github.com/elastic/kibana/blob/master/packages/kbn-es-query/src/es_query/build_es_query.js#L46)?